gateway-queue/src/lib.rs

```diff
@@ -156,12 +147,10 @@ impl Queue for LocalQueue {
             let (tx, rx) = oneshot::channel();
 
             if let Err(_source) = self.0.send(tx) {
-                #[cfg(feature = "tracing")]
                 tracing::warn!("skipping, send failed: {:?}", _source);
                 return;
             }
 
-            #[cfg(feature = "tracing")]
             tracing::info!("shard {}/{} waiting for allowance", _id, _total);
```

Comment:
And also in the method's arguments
```suggestion
            if let Err(source) = self.0.send(tx) {
                tracing::warn!("skipping, send failed: {:?}", source);
                return;
            }

            tracing::info!("shard {}/{} waiting for allowance", id, total);
```